README.rst

```diff
@@ -6,33 +6,40 @@ A Python native, OS native GUI toolkit.
 Prerequisites
 ~~~~~~~~~~~~~
 
-Toga has some minimum requirements:
+Minimum requirements
+^^^^^^^^^^^^^^^^^^^^
 
 * Toga requires **Python 3**. Python 2 is not supported.
 
 * If you're on macOS, you need to be on 10.7 (Lion) or newer.
 
 * If you're on Linux, you need to have GTK+ 3.10 or later. This is the version
   that ships starting with Ubuntu 14.04 and Fedora 20. You also need to install
-  the Python 3 bindings to GTK+. If you want to use the WebView widget, you'll
-  also need to have WebKit, plus the GI bindings to WebKit installed. This means
-  you'll need to install the following:
-
+  the Python 3 bindings to GTK+.
+  
+* We're working on Windows support, but not all features and widgets are supported. At a minimum, you'll need Python 3 and .NET Framework 4. This has been tested on Windows 10, but should work on 7 and 8. Pull requests, help and corrections are most welcome.
+  
+Optional extras
+^^^^^^^^^^^^^^^
+
+If you want to use the WebView widget, you'll
+also need to have WebKit, plus the GI bindings to WebKit installed. This means
+you'll need to install the following:
   * **Ubuntu 14.04** ``apt-get install python3-gi gir1.2-webkit-3.0``
 
-  * **Ubuntu 16.04** ``apt-get install python3-gi gir1.2-webkit-4.0``
+  * **Ubuntu 16.04 / Debian 8 ** ``apt-get install python3-gi gir1.2-webkit-4.0``
```

Comment:
In RST, you can't have any spaces between the end of a word and the end of the bold syntax. Remove the space between `8` and `**` here, and it will display correctly.